### PR TITLE
Fix segfault due to non-portable forest build

### PR DESCRIPTION
**Pull request by elmattic**
## Summary of changes

Changes introduced in this pull request:

-

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

**Comment by elmattic** [comment]
It's all green ✅.

**Comment by elmattic** [comment]
It's all green again ✅.

**Comment by elmattic** [comment]
It's all green again ✅.
But build was done using a non AVX512 CPU (Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz).

**Comment by elmattic** [comment]
Relevant link:
https://github.com/supranational/blst/blob/master/bindings/rust/README.md#build

It's all green now ✅.

Build of Forest daemon was done on Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz.
Run was done on Intel(R) Xeon(R) CPU E5-2673 v3 @ 2.40GHz (so without the ADX instruction set).

**Review by LesnyRumcajs — APPROVED** [review]
Fantastic job!

**Review by hanabi1224 — APPROVED** [review]
Worth another release for this fix? : )

File: Cargo.toml [hunk 1]
```diff
@@ -58,7 +58,7 @@ axum = "0.6"
 backoff = "0.4"
 base64 = "0.21"
 blake2b_simd = "1.0"
-bls-signatures = { version = "0.13", default-features = false, features = ["blst"] }
+bls-signatures = { version = "0.13", default-features = false, features = ["blst", "blst-portable"] }
```
[the remark this answers is not in the document]
Comment (aatifsyed) [reply Cargo.toml:61]: worth a comment inline too

[the remark this answers is not in the document]
Comment (elmattic) [reply Cargo.toml:61]: https://github.com/ChainSafe/forest/pull/2892

**Pull request merged**